sdks/python/apache_beam/typehints/row_type.py

```diff
@@ -112,8 +109,19 @@ def from_user_type(
     return None
 
   @staticmethod
-  def from_fields(fields: Sequence[Tuple[str, type]]) -> RowTypeConstraint:
-    return RowTypeConstraint(fields=fields, user_type=None)
+  def from_fields(
+      fields: Sequence[Tuple[str, type]],
+      schema_id: Optional[str] = None,
+      schema_options: Optional[Sequence[Tuple[str, Any]]] = None,
+      field_options: Optional[Dict[str, Sequence[Tuple[str, Any]]]] = None,
+      schema_registry: SchemaTypeRegistry = None,
```

Comment:
Would this fix this lint issue
```
18:16:14 Running flake8...
18:16:24 apache_beam/typehints/row_type.py:116:24: F821 undefined name 'SchemaTypeRegistry'
18:16:24       schema_registry: SchemaTypeRegistry = None,
18:16:24                        ^
18:16:24 apache_beam/typehints/row_type.py:183:24: F821 undefined name 'SchemaTypeRegistry'
18:16:24       schema_registry: SchemaTypeRegistry = None,
18:16:24                        ^
18:16:24 2     F821 undefined name 'SchemaTypeRegistry'
```

```suggestion
      schema_registry: Optional[SchemaTypeRegistry] = None,
```